src/main/java/org/example/camunda/process/solution/MyProcessVariables.java

```diff
@@ -0,0 +1,20 @@
+package org.example.camunda.process.solution;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@Getter
+@Setter
+@Accessors(chain = true)
+public class MyProcessVariables extends ProcessVariables {
+
+  private String businessKey;
+  private Boolean result;
+  private String status;
+  private String subject;
+  private String email;
+  private String message;
+  private String caseId;
+  private String searchTerm;
```

Comment:
Let's move these into the existing class

Reply:
I tried it in my current PoC and it's great. Adding variables becomes so much easier!